standalone/server.c

```diff
@@ -998,26 +998,14 @@ AP_DECLARE(void) unixd_pre_config(apr_pool_t *ptemp)
     apr_finfo_t wrapper;
 
 #if AP_SERVER_MAJORVERSION_NUMBER > 1 && AP_SERVER_MINORVERSION_NUMBER < 3
-#ifdef WAF_JSON_LOGGING_ENABLE
```

Comment:
Could you add a new commit for WAF_JSON_LOGGING_ENABLE

Reply:
Sure